featurebyte/api/aggregator/forward_aggregator.py

```diff
@@ -33,6 +34,8 @@ def forward_aggregate(
         method: str,
         window: Optional[str] = None,
         target_name: Optional[str] = None,
+        fill_value: OptionalScalar = None,
+        skip_fill_na: bool = False,
```

Comment:
You many need to update the forward aggregator feature definition here: https://github.com/featurebyte/featurebyte/blob/0edb6809677c3214036daa893bab30dc3e21a4f8/featurebyte/query_graph/node/generic.py#L553-L559